docs/sources/operators-guide/deploying-grafana-mimir/jsonnet/configuring.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Configuring Grafana Mimir deployment with Jsonnet"
+menuTitle: "Configuring deployment"
+description: "Learn how to configure Grafana Mimir when using Jsonnet."
+weight: 20
+---
+
+# Configuring Grafana Mimir deployment with Jsonnet
+
+Notable features of the Mimir Jsonnet are described here in detail.
+To learn how to get started, see [Deploying Grafana Mimir with Jsonnet and Tanka]({{< relref "./deploying.md" >}}).
+
+## Anti-affinity
+
+Given the distributed nature of Mimir, both performance and reliability are improved when pods are spread across different nodes.
+For example, multiple queriers can be processing the same query at the same time, so it's better to distribute them across different nodes. Morever, since losing multiple ingesters can cause data loss, it's also important to have them spread.
+
+For this reason, by default, anti-affinity rules are applied to some Deployments and StatefulSets.
+These anti-affinity rules can become an issue when playing with Mimir in a single-node Kubernetes cluster, so anti-affinity can be disabled by setting the configuration values `_config.<component>_allow_multiple_replicas_on_same_node`.
+
+### Example: disable anti-affinity
+
+```jsonnet
+local mimir = import 'mimir/mimir.libsonnet';
+
+mimir {
+  _config+:: {
+    // ... configuration values
+    distributor_allow_multiple_replicas_on_same_node: true,
+    ingester_allow_multiple_replicas_on_same_node: true,
+    ruler_allow_multiple_replicas_on_same_node: true,
+    querier_allow_multiple_replicas_on_same_node: true,
+    query_frontend_allow_multiple_replicas_on_same_node: true,
+    store_gateway_allow_multiple_replicas_on_same_node: true,
+  },
+}
+```
+
+## Resources
+
+Default scaling of Mimir components in the provided Jsonnet is opinionated and based on years of experience running it at Grafana Labs.
+The default resources requests and limits are also fine-tuned for the [provided alerting rules]({{< relref "../../visualizing-metrics/_index.md" >}}).
```

Comment:
```suggestion
The default resource requests and limits are also fine-tuned for the provided alerting rules. For more information, see [Visualizing metrics]({{< relref "../../visualizing-metrics/_index.md" >}}).
```

Reply:
> This URL and page title are off. Please change the title of that _index.md file from "Monitoring metrics" to "Visualizing metrics".

I think we need to change the URL, not the title. The title is more correct than the URL. I will create an issue about it.